plugin/file/secondary.go

```diff
@@ -170,11 +191,17 @@ Restart:
 			if ok {
 				if err := z.TransferIn(); err != nil {
 					// transfer failed
+					UpdatedSerial.WithLabelValues(z.origin).Dec()
+					UpdatedSuccess.WithLabelValues(z.origin).Dec()
```

Comment:
these all miss the server label

Reply:
Going further, I don't see a way to actually add the correct `server` label, so it should be removed from all of the metrics here, and just have the origin.